tests/cases/fourslash/navigationBarAnonymousClassAndFunctionExpressions.ts

```diff
@@ -91,6 +91,17 @@ verify.navigationBar([
         "indent": 2
     },
     {
+        "text": "y",

```

Comment:
does the associated comment on the source line need to change? Currently it says "this is not a top-level function".